compliance_controller/compliance_control_msgs/msg/CompliantVelocities.msg

```diff
@@ -0,0 +1,5 @@
+# This array holds velocities for each joint
+std_msgs/Float64MultiArray compliant_velocities
```

Comment:
I'm just a user here, so this is me being curious: what is the reason you chose to use a `Float64MultiArray` here? Would something like this not work:

```yaml
float64[] velocity
```
perhaps even with a `string[] name` to be able to match velocities with joints actually controlled by the controller?

`MultiArray` allows for n-dim arrays though: is that (part of) the rationale?

Reply:
The only reason was to stay compatible with a JointGroupVelocity controller... but yeah, I think it should contain joint names, too.